controller/array_action/array_mediator_xiv.py

```diff
@@ -367,6 +367,22 @@ def delete_snapshot(self, snapshot_id):
 
         logger.info("Finished snapshot deletion. id : {0}".format(snapshot_id))
 
+    def _get_cli_host_by_name(self, host_name):
```

Comment:
```suggestion
    def _get_cli_host(self, host_name):
```
I think it's clear enough from the parameter name

Reply:
we have `get_host_by_host_identifiers(self, initiators)`, it could be clear enough as well, should we find other names?

Reply:
maybe `get_host_definition`? not mandatory for this PR though

Reply:
done